### ISSUE TITLE
Documentation parameter ptr is mismatched of bindSymbol_stdcall

### ISSUE THREAD

**Issue by foerdi** [BindBC/bindbc-loader#4]
The parameter `ptr` in the documentation of the function `bindSymbol_stdcall` is mismatched https://github.com/BindBC/bindbc-loader/blob/b302fbb52c67504579e9f7a064090aee71cd459d/source/bindbc/loader/sharedlib.d#L119

I got the warning/error: 
```bash
.dub/packages/bindbc-loader-0.2.1/bindbc-loader/source/bindbc/loader/sharedlib.d(119,7): Warning: Ddoc: function declaration has no parameter 'ptr'
```

**Comment by mdparker** [comment]
Sorry for the late response. I completely missed this. So this is a Ddoc error then?

**Comment by mdparker** [comment]
Should be fixed in 0.3.1, which I'll be pushing out as soon as I figure out how to handle #5
